site2/docs/developing-binary-protocol.md

```diff
@@ -63,6 +63,14 @@ It is created by the broker when the message arrived at the broker and passed wi
 | `broker_timestamp` | Optional        | The timestamp when a message arrived at the broker (`id est` as the number of milliseconds since January 1st, 1970 in UTC)      |
 | `index`            | Optional        | The index of the message. It is assigned by the broker.
 
+If you want use broker entry metadata for **brokers**, configure the [`brokerEntryMetadataInterceptors`](reference-configuration.md#broker) parameter in the `broker.conf` file.
+
+If you want use broker entry metadata for **consumers**:
+
+1. Use the client protocol version [no later than 18](https://github.com/apache/pulsar/blob/ca37e67211feda4f7e0984e6414e707f1c1dfd07/pulsar-common/src/main/proto/PulsarApi.proto#L259).
+   
```

Comment:
could we change `no later than 18` to `18 or higher`

Reply:
make sense. I prefer `18 or later`, reason: https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.tdt6lehtxogm